bigquery/google/cloud/bigquery/client.py

```diff
@@ -611,6 +611,11 @@ def load_table_from_storage(self, source_uris, destination,
         :type job_id: str
         :param job_id: (Optional) Name of the job.
 
+        :type job_id_prefix: str or ``NoneType``
```

Comment:
remove the first "job ID:"

"the user-provided prefix for a ..."

Reply:
Fixed with c0a7120